web/src/components/TickerTable/index.tsx

```diff
@@ -41,21 +45,21 @@ export const TickerTable: React.FC<Props> = ({
                     </td>
                     <td>
                         <span>
-                            <Decimal fixed={market.amount_precision} thousSep=",">
+                            <Decimal fixed={!isFixedPrecision ? market.amount_precision : market.price_precision} thousSep=",">
                                 {market.high}
                             </Decimal>
                         </span>
                     </td>
                     <td>
                         <span>
-                            <Decimal fixed={market.amount_precision} thousSep=",">
+                            <Decimal fixed={!isFixedPrecision ? market.amount_precision : market.price_precision} thousSep=",">
                                 {market.low}
                             </Decimal>
                         </span>
                     </td>
                     <td>
                         <span>
-                            <Decimal fixed={market.amount_precision} thousSep=",">
+                            <Decimal fixed={!isFixedPrecision ? market.amount_precision : FIXED_PRECISION} thousSep=",">
```

Comment:
market.volume is a value in quote currency. that's why here we also put market.price_precision